pandas/tests/extension/base/constructors.py

```diff
@@ -126,6 +126,12 @@ def test_construct_empty_dataframe(self, dtype):
     def test_empty(self, dtype):
         cls = dtype.construct_array_type()
         result = cls._empty((4,), dtype=dtype)
-
         assert isinstance(result, cls)
         assert result.dtype == dtype
+        assert result.shape == (4,)
+
+        # GH#19600 method on ExtensionDtype
+        result2 = dtype.empty((4,))
+        assert isinstance(result2, cls)
+        assert result2.dtype == dtype
+        assert result2.shape == (4,)
```

Comment:
If we make this public, we should maybe a test that passing a scalar integer works as well (if we want to allow that, following np.empty)